lib/init/grass7.html

```diff
@@ -110,6 +110,23 @@ <h2>DESCRIPTION</h2>
 these checks fail then <em>grass79</em> will automatically switch back
 to the text user interface mode.
 
+<h2>FLAGS</h2>
+
+The flag <b>--config option</b> prints GRASS GIS configuration and
+version parameters, with the options:
+
+<ul>
+<li><b>arch</b>: system architecture (e.g., <tt>x86_64-pc-linux-gnu</tt>)</li>
+<li><b>build</b>:  (e.g., <tt>./configure  --with-cxx --enable-largefile --with-proj [...]</tt>)</li>
+<li><b>compiler</b>:  (e.g., <tt>gcc</tt>)</li>
+<li><b>date</b>:  (e.g., <tt>Tue Mar 31 20:34:57 2020 +0200</tt>)</li>
+<li><b>path</b>:  (e.g., <tt>/usr/lib64/grass78</tt>)</li>
+<li><b>revision</b>:  (e.g., <tt>745ee7ec9</tt>)</li>
+<li><b>svn_revision</b>:  (e.g., <tt>062bffc8</tt>)</li>
+<li><b>version</b>:  (e.g., <tt>7.9.dev</tt>)</li>
```

Comment:
It this PR work in progress or ready for review? (I see two spaces where you have "system architecture" above.)

Reply:
fixed